public/css/main.css

```diff
@@ -1,15 +1,13 @@
 ﻿/*icons*/
 @font-face {
   font-family: 'icomoon';
-  src:  url('/fonts/icomoon.eot?g003m6');
-  src:  url('/fonts/icomoon.eot?g003m6#iefix') format('embedded-opentype'),
-    url('/fonts/icomoon.ttf?g003m6') format('truetype'),
-    url('/fonts/icomoon.woff?g003m6') format('woff'),
-    url('/fonts/icomoon.svg?g003m6#icomoon') format('svg');
+  src: url('/fonts/icomoon.eot?g003m6');
+  src: url('/fonts/icomoon.eot?g003m6#iefix') format('embedded-opentype'), url('/fonts/icomoon.ttf?g003m6') format('truetype'), url('/fonts/icomoon.woff?g003m6') format('woff'), url('/fonts/icomoon.svg?g003m6#icomoon') format('svg');
```

Comment:
Why is this being compacted?  It's minimal byte savings but it's far less legible.